views/includes/page-templates/beneficial-owner-individual.html

```diff
@@ -23,25 +23,13 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
 
       {% set dateText = "When did they become a beneficial owner for the overseas entity?" %}
       {% include "includes/inputs/date/start-date-input.html" %}
+      {% if entity_number %}
+        {% include "includes/inputs/fields/is-still-a-beneficial-owner-input.html" %}
+      {% endif %}
 
-      {% call govukFieldset({
-        legend: {
-          text: "What is their nature of control?",
-          classes: "govuk-fieldset__legend--m",
-          isPageHeading: false
-        }
-      }) %}
-
-      {% set errorMessage = errors.beneficial_owner_nature_of_control_types if errors %}
-      {% include "includes/inputs/nature-of-control-checkboxes/beneficial-owner-nature-of-control-types.html" %}
-
-      {% set errorMessage = {} if errors.beneficial_owner_nature_of_control_types %}
-      {% include "includes/inputs/nature-of-control-checkboxes/trustees-nature-of-control-types.html" %}
-
-      {% set errorMessage = {} if errors.beneficial_owner_nature_of_control_types %}
-      {% include "includes/inputs/nature-of-control-checkboxes/non-legal-firm-nature-of-control-types.html" %}
-
-      {% endcall %}
+      {% set isBeneficialOwnerGov = false %}
+      {% set natureOfControlText = "What is their nature of control?" %}
+      {% include "includes/page-templates/beneficial-owner-nature-of-control-types.html" %}
```

Comment:
This template didn't come through on registration for both the BO and MO

Reply:
Just to clarify, "previous answer" is the question that determines which MO/BO add options are shown. Went over it on a call with Olayode and all is good. All NOC sections show up as expected for all 3 BO types.